h5p-bildetema/src/components/CollectionsController/CollectionController.tsx

```diff
@@ -15,11 +15,11 @@ import { useL10ns } from "../../hooks/useL10n";
 import { H5PWrapper } from "../../h5p/H5PWrapper";
 
 type CollectionControllerProps = {
-  rtl: boolean;
+  rtl?: boolean;
```

Comment:
The null checker might be problematic? What was the reason behind having a nullchecker for the boolean value?

Reply:
Make sure to remove it from  rtl={rtl}

Reply:
That component (SubHeader) is used somewhere else where the rtl prop is used